src/GenAPI/Microsoft.DotNet.GenAPI/SyntaxGeneratorExtensions.cs

```diff
@@ -92,5 +95,20 @@ private static ConstructorInitializerSyntax GenerateBaseConstructorInitializer(I
 
             return constructorInitializer;
         }
+
+        private static BaseListSyntax? GetBaseTypeList(this SyntaxGenerator syntaxGenerator,
+            INamedTypeSymbol type,
+            ISymbolFilter symbolFilter)
+        {
+            List<BaseTypeSyntax> baseTypes = new();
+
+            if (type.TypeKind == TypeKind.Class && type.BaseType != null && symbolFilter.Include(type.BaseType))
+            {
+                baseTypes.Add(SyntaxFactory.SimpleBaseType((TypeSyntax)syntaxGenerator.TypeExpression(type.BaseType)));
+            }
+
+            baseTypes.AddRange(type.Interfaces.Where(symbolFilter.Include).Select(i => SyntaxFactory.SimpleBaseType((TypeSyntax)syntaxGenerator.TypeExpression(i))));
+            return baseTypes.Count == 0 ? null : SyntaxFactory.BaseList(SyntaxFactory.SeparatedList(baseTypes));
```

Comment:
Style nit. Start with the positive case first and wrap into multiple lines for readability.

```suggestion
            return baseTypes.Count > 0 ?
                SyntaxFactory.BaseList(SyntaxFactory.SeparatedList(baseTypes)) :
                null;
```

Reply:
Thanks, updated